core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1590,7 +1590,7 @@ private SqlLibraryOperators() {
   @LibraryOperator(libraries = {ORACLE})
   public static final SqlFunction ORACLE_NEXT_DAY =
       new SqlFunction(
-          "NEXT_DAY",
+          "ORACLE_NEXT_DAY",
```

Comment:
This approach probably needs to be discussed again with the stakeholders @dm-shantanuc , but will merge for now. Create a separate Jira to handle this better.